pootle/apps/pootle_language/forms.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django import forms
+from django.utils.translation import ugettext_lazy as _
+
+from .models import Language
+
+
+class LanguageSpecialCharsForm(forms.ModelForm):
+
+    class Meta:
+        model = Language
+        fields = ('specialchars',)
+        labels = {
+            'specialchars': _('Enter any special characters that users might '
```

Comment:
Sorry, probably I didn't express properly what I meant with the fact that the layout can be changed (no need to duplicate strings):
```diff
diff --git a/pootle/apps/pootle_language/forms.py b/pootle/apps/pootle_language/forms.py
index b222a2b..494bb40 100644
--- a/pootle/apps/pootle_language/forms.py
+++ b/pootle/apps/pootle_language/forms.py
@@ -18,7 +18,3 @@ class LanguageSpecialCharsForm(forms.ModelForm):
     class Meta:
         model = Language
         fields = ('specialchars',)
-        labels = {
-            'specialchars': _('Enter any special characters that users might '
-                              'find difficult to type'),
-        }
diff --git a/pootle/templates/languages/admin/characters.html b/pootle/templates/languages/admin/characters.html
index c0e7122..bea3675 100644
--- a/pootle/templates/languages/admin/characters.html
+++ b/pootle/templates/languages/admin/characters.html
@@ -12,7 +12,8 @@
     {{ form.non_field_errors }}
     {% for field in form %}
     <p>{{ field.errors }}
-      {{ field.label_tag }} {{ field }}
+      <label for="{{ field.id_for_label }}">{{ field.help_text }}</label>
+      {{ field }}
     </p>
     {% endfor %}
     <p class="buttons">
```